### DISCUSSION TITLE
The ideal "admin ontology" page in OntoPortal

### DISCUSSION THREAD

**Discussion by jonquet** [ontoportal/ontoportal-project#20] [Ideas]
In preparation of setting up an "admin ontology" functionality and corresponding page, I like to brainstorm about the possible features such a page would ideally offer (i.e., not necessarily doable today).

What is envisioned is that an OntoPortal logged-in user with admin access to a given ontology could access a specific page provided by OntoPortal (similar to the Edit submission one) to decide how to setup/configure OntoPortal for this ontology only. This page would allow him/her to activate or not some functionalities OntoPortal offers, to see the parsing reports, admin the versions more closely, maybe call external tools, etc.  

Please share your thoughts!

**Comment by jonquet** [comment]
Here those are mine: 
- Declare the design properties (prefLabel, synonyms, author, dates, etc.)
- Decide to show obsolete content or not
- Enable OFAIRe
- Enable Notes
- Enable Reviews
- Enable issue push to GitHub (need metadata)
- Choose root resources for hierarchy (need metadata)
- Display part-of hierarchy (need metadata)
- Manage DOI request
- Moderate the forum
- Admin user access
- Invite collaborators button
- See parsing logs
- Call external services (mappings, oops, foops, oak, etc)
- Curate merged mappings 
- Declare projects using the ontology
- See who used the ontology in the UI
- See who used the ontology with the API
- Admin the database links (assuming we have re-implemented something to connect ontologies to external databases)
- Admin link to editor (WebProtege or VocBench)
- Pushing ontology to ORCID (contribution)
- Tweet about things

**Reply by graybeal** [reply]
> What is envisioned is that a user with admin access to an ontology could decide how to setup OntoPortal for this ontology only, access reports, connect to other 3rd party tools, etc.

Can you please describe this in a little more detail? Is that idea that I could set a configuration file in GitHub, and OntoPortal would read the config file and change the behaviors it presents to a user (the ontology owner? administrators?) any logged-in user? any public user?) according to those settings?  If so, why would this be better placed in GitHub than in OntoPortal?

**Reply by jonquet** [reply]
Ok I will precise. GitHub is mostly off topic here. I meant an admin page for the ontology inside the portal.

**Comment by syphax-bouazzouni** [comment]
Some other thoughts here https://github.com/agroportal/project-management/issues/452
